mobile_verifier/src/heartbeats/mod.rs

```diff
@@ -414,16 +415,27 @@ impl ValidatedHeartbeat {
         })
     }
 
-    pub async fn write(&self, heartbeats: &FileSinkClient) -> file_store::Result {
+    pub async fn write(
+        &self,
+        max_distance_to_asserted: u32,
+        heartbeats: &FileSinkClient,
+    ) -> file_store::Result {
+        let location_trust_score_multiplier = self.cell_type.location_weight(
```

Comment:
Maybe we should persist the `location_trust_score_multiplier` into the database.  That way we can ensure that the average calculation uses the same score that was recorded with the heartbeat.